### ISSUE TITLE
Can I use this without the textarea input?

### ISSUE THREAD

**Issue by TroyCoombs** [angular-material-extensions/link-preview#105]
I would like to use this in a chat application and simply pass it the message in a message thread to display a link-preview if the message contains a url.  Can I do that?

**Comment by shekar-raja** [comment]
Did you get any solution?

**Comment by TroyCoombs** [comment]
nope.

**Comment by shekar-raja** [comment]
Did you figure out how to do it?

**Comment by TroyCoombs** [comment]
no, not using this extension.  I've been side tracked on implementing this but my intent is to just build it from scratch unfortunately.

**Comment by hvsharma63** [comment]
I've had the same issue, I had to do it in the back-end to display the previewed content.
